src/UI/templates/default/Button/tpl.primary.html

```diff
@@ -0,0 +1 @@
+<button type="button" class="btn btn-default btn-primary" href="{ACTION}" {DISABLED}>{LABEL}</button>
```

Comment:
Thank you for your feedback. 

See my comment in the above discussion on a similar matter in regard to the blocks in the glyph template. Generic templates pose issues for designers.